File: src/SOP8/광양/EgovLogin/base-account-ibll/src/main/java/egovframework/base/account/ibll/response/LoginResult.java
package egovframework.base.account.ibll.response;

import egovframework.base.account.ibll.models.ApplicationUser;
import egovframework.base.response.MessageResult;

public class LoginResult extends MessageResult {

    private ApplicationUser user = null;

    public ApplicationUser getUser() {
        return user;
    }

    public void setUser(ApplicationUser user) {
        this.user = user;
    }

    public LoginResult() {
        super();
    }

    public LoginResult(boolean success, String message) {
        super(success, message);
    }
}

File: src/SOP8/Base/Account/Base.Account.IBLL/Response/LoginResult.cs
using Response;
using Base.Model.Account;

namespace Base.Account.IBLL.Response
{
    using Models;

    public class LoginResult : MessageResult
    {
        private ApplicationUser m_user = null;

        public ApplicationUser User
        {
            get { return m_user; }
            set { m_user = value; }
        }

        public LoginResult()
            : base()
        {
        }

        public LoginResult(bool success, string message)
            : base(success, message)
        {
        }
    }
}
